### PR TITLE
Neyyi karam dosa

### PR DESCRIPTION
**Pull request by joindevopscloud**

### CODE REVIEW

File: menu.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 dosa
-karam
-oil
\ No newline at end of file
+extra karam
```
Comment (sivakumarreddym) [inline menu.txt:2]: do we really require extra karam?

Comment (joindevopscloud) [reply menu.txt:2]: Yes, if extra karam extra tasty

Comment (sivakumarreddym) [reply menu.txt:2]: yaa, that's nice

**Review by sivakumarreddym — APPROVED** [review]
yes neyyi karam is good

**Pull request merged**